src/Data/Config.php

```diff
@@ -85,7 +85,7 @@ public function graphql_wp_query_cursor_pagination_support( $where, \WP_Query $q
 							$order_compare = ( 'ASC' === $order ) ? '>' : '<';
 							$value = $cursor_post->{$by};
 							if ( ! empty( $by ) && ! empty( $value ) ) {
-								$where .= $wpdb->prepare( " AND {$wpdb->posts}.{$by} {$order_compare} '{$value}'", $value );
+								$where .= $wpdb->prepare( " AND {$wpdb->posts}.{$by} {$order_compare} '%s'", $value );
```

Comment:
I'm not sure you actually have to add the placeholder within the quotes here? Does it not work without them?

Reply:
@CodeProKid let me check. I think you're probably right